MapboxNavigation/Constants.swift

```diff
@@ -35,3 +35,13 @@ public let MBCongestionAttribute = "congestion"
  The minimum volume for the device before a gentle warning is emitted when beginning navigation.
  */
 public let NavigationViewMinimumVolumeForWarning: Float = 0.3
+
+extension Notification.Name {
+    
+    /**
+     Posted when `StyleManager` applies a style that was triggered by change of time of day, or when entering or exiting a tunnel.
```

Comment:
This comment should refer to `StyleManagerDelegate.styleManager(_:didApply:)` as being equivalent and vice versa.